src/HomeWork/StudentGrade.java

```diff
@@ -0,0 +1,41 @@
+package HomeWork;
+
+import java.util.Scanner;
+import java.util.TreeMap;
+
+public class StudentGrade {
+    public static void main(String[] args) {
+        boolean isExit;
+        do {
+            try {
+                TreeMap<Integer, String> gradeMap = new TreeMap<>();
+                gradeMap.put(90, "A");
+                gradeMap.put(80, "B");
+                gradeMap.put(70, "C");
+                gradeMap.put(60, "D");
+                gradeMap.put(0, "E");
+
+                Scanner scanner = new Scanner(System.in);
+                System.out.print("Enter the student's score: ");
+                int score = scanner.nextInt();
+
+                String grade = determineGrade(gradeMap, score);
```

Comment:
I think you need to add validation if the entered number is in the limits: 0-100 and throw an exception if it is not